setup.py

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env python3
+
+# Core
+from setuptools import setup, find_packages
+
+setup(
+    name='canonicalwebteam.http',
```

Comment:
The package name should just be called `canonicalwebteam`, which also suggests that this repo should be called `canonicalwebteam`. `http` is just a module within the `canonicalwebteam` package.

Reply:
There [is a (possibly somewhat loose) recommendation](https://legacy.python.org/dev/peps/pep-0423/#top-level-namespace-relates-to-code-ownership) to use namespaces in PyPi module names to denote ownership.

We therefore use `canonicalwebteam` is a namespace, for collecting our PyPi packages - e.g. [gsa](https://github.com/canonical-webteam/gsa), [yaml-redirect](https://github.com/canonical-webteam/yaml-redirects) etc. These can exist in harmony through [packaging namespace packages](https://packaging.python.org/guides/packaging-namespace-packages/), and for that pattern you don't mention the namespace package in `setup.py`, only the child module.